lib/ember-cli/app.rb

```diff
@@ -339,7 +339,7 @@ def env_hash
 
     def exec(cmd, method: :system)
       Dir.chdir root do
-        Kernel.public_send(method, env_hash, cmd, err: :out)
+        Kernel.public_send(method, env_hash, cmd, err: :out) or fail(cmd)
```

Comment:
I think you might also be able to use `Kernel.exit` here: https://github.com/thoughtbot/parity/blob/256f9a3f995d21fa6059efcd523855f2a1a0c381/bin/production#L8